tests/functional/flakes/shebang.sh

```diff
@@ -69,7 +69,7 @@ EOF
 cat >> "$scriptDir/shebang-inline-expr.sh" <<"EOF"
 #! nix --offline shell
 #! nix --impure --expr ``
-#! nix let flake = (builtins.getFlake (toString ../flake1)).packages;
```

Comment:
Maybe leave this test and add this as a second one so it ensures the backwards compatibility?